centos-7-and-extras-x86_64.cfg

```diff
@@ -45,4 +45,13 @@ failovermethod=priority
 exclude=docker-python python-docker-py
 gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-7
 gpgcheck=1
+
+[openshift]
+name=openshift
```

Comment:
ditto.

Reply:
I don't think it's needed in the base buildroot.

Reply:
Will remove.